docs/rm/snax_cluster.md

```diff
@@ -0,0 +1,198 @@
+    * `data_arga` is the CSR address `0x3c0`
+    * `data_argb` contains the contents inside register `r1`
+    * `data_argc` is unused for SNAX.
+* In the response channel:
+    * `id` returned comes from the original request which was `r3`.
+    * `error` is 0 but can be any number depending on response.
+    * `data` is whatever the accelerator returns back. It's possible that write only requests don't need to return anything back.
+
+
+### Snitch TCDM Ports
+
+The Snitch TCDM ports function similarly like the accelerator request-response ports. The figure below shows the TCDM connection:
+
+![SNAX TCDM ports](https://drive.google.com/uc?id=1GWrK3lolX0v89iy3oNbgR8guhpaXnRFa)
+
+It also uses the same valid-ready handshake similar to the accelerator ports. The signal definitions for the request port are:
+
+| Signal            | Description                                                                                                   | 
+| ----------------- | ------------------------------------------------------------------------------------------------------------  |
+| q_valid           | Request side valid signal.                                                                                    |
+| write             | Write signal. 1 means to write.                                                                               |
+| addr              | Memory address to write to.                                                                                   |
+| amo               | Atomic memory operation. Details are in [Request and Response](reqrsp_interface.md) section                   |
+| data              | The data to be written.                                                                                       |
+| user              | User field pertains to which core is accessing the port.                                                      |
+| strb              | Byte masking for data writes                                                                                  |
+
+The signal definitions for the response port are:
+
+| Signal            | Description                                                                                                   | 
+| ----------------- | ------------------------------------------------------------------------------------------------------------  |
+| p_valid           | Response side valid signal.                                                                                   |
+| q_ready           | Request side ready signal.                                                                                    |
+| data              | The read data to be returned to the core.                                                                     |
+
+Notice that the `q_ready` signal of the TCDM was placed in the response ports. This is just to indicate that the direction is from the TCDM interconnect towards the Snitch core.
```

Comment:
Shall we add as there is no `p_ready` signal, so the accelerator needs to make sure to catch every `p_valid` signal other some data transaction might be lost. (The `p_valid` only asserts in one cycle, the request module does not check if the receiver is ready)

Reply:
Adding this description wait.